src/EPR.PRN.Backend.API/Handlers/UpdateRegulatorRegistrationTaskHandler.cs

```diff
@@ -0,0 +1,13 @@
+using EPR.PRN.Backend.API.Commands;
+using EPR.PRN.Backend.Data.DataModels.Registrations;
+using EPR.PRN.Backend.Data.Interfaces.Regulator;
+
+namespace EPR.PRN.Backend.API.Handlers;
+
+public class UpdateRegulatorRegistrationTaskHandler : UpdateRegulatorTaskHandlerBase<UpdateRegulatorRegistrationTaskCommand, IRegulatorRegistrationTaskStatusRepository, RegulatorRegistrationTaskStatus>
+{
+    public UpdateRegulatorRegistrationTaskHandler(IRegulatorRegistrationTaskStatusRepository repository)
+        : base(repository)
+    {
+    }
```

Comment:
Could use a primary constructor here instead perhaps?
```suggestion
public class UpdateRegulatorRegistrationTaskHandler(IRegulatorRegistrationTaskStatusRepository repository)
    : UpdateRegulatorTaskHandlerBase<UpdateRegulatorRegistrationTaskCommand, IRegulatorRegistrationTaskStatusRepository,
        RegulatorRegistrationTaskStatus>(repository);
```